release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -246,7 +246,7 @@ For more information about the dynamic plug-in, see xref:../web_console/dynamic-
 
 [id="ocp-4-10-multicluster-console-technology-preview"]
 ==== Multicluster console (Technology Preview)
-Starting with {product-title} 4.10, the multicluster console is now available as a Technology Preview feature. By enabling this feature, you can connect to remote clusters' API servers from a single {product-title} console.
+Starting with {product-title} 4.10, the multicluster console is now available as a Technology Preview feature. By enabling this feature, you can connect to remote clusters' API servers from a single {product-title} console. You must have Red Hat Advanced Cluster Management (ACM) installed to enable this feature.
```

Comment:
I think we want to mention MCE as an alternative, although the operator is not yet available. cc @showeimer

Reply:
Should we include this for now and then mention MCE when it becomes available with ACM 2.5? @spadgett @showeimer